Dependencies/xr/Source/ARCore/XR.cpp

```diff
@@ -34,32 +34,27 @@
 using namespace android;
 using namespace android::global;
 
-namespace xr
-{
+namespace xr {
```

Comment:
It looks like you got attacked by a rogue clang formatter. :smiley: If you didn't intend to change this file (or, at least, not this substantially), can you revert the unintended changes to make it easier to identify the intended ones?

Reply:
this should hopefully now be fixed